src/components/User/Users/components/UserList.vue

```diff
@@ -143,35 +131,23 @@ import { Sdk } from '@/sdk'
 
 import 'mdi-vue/DownloadIcon'
 
-const VIEW_MODES_VERBOSE = Object.freeze({
-  index: 'index',
-  user: 'user',
-})
-
 export default {
   components: {
     SelectField,
     InputField,
-    UserView,
     AccountStateGetter,
     CollectionLoader,
   },
 
   data () {
     return {
-      VIEW_MODES_VERBOSE,
       filters: {
         role: '',
         requestor: '',
       },
-      view: {
-        mode: VIEW_MODES_VERBOSE.index,
-        userId: null,
-        scrollPosition: 0,
-      },
+      userId: null,
```

Comment:
Useless

Reply:
Also, `'filters.state'` useless too in the `watch` hook

Reply:
'filters.state' came with automerge master to my branch. It is Maksym's feature. Should I realy delete this?